### PR TITLE
4.4 rule fixes

### PR DESCRIPTION
**Pull request by yapei**
@SSshahan  Can you help review? Thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.1/deployment.xyaml [hunk 1]
```diff
@@ -45,3 +45,11 @@ delete_dc_with_dependency_objects:
     button_text: Delete
   action: click_one_dropdown_action
   action: click_button
+check_sections_on_container_details:
```
Comment (yapei) [inline lib/rules/web/admin_console/4.1/deployment.xyaml]: In 4.4, `Container Overview` title is changed to `Container Details`
Screen Shot 2020-02-11 at 4 52 49 PM

File: lib/rules/web/admin_console/4.4/overview.xyaml [hunk 2]
```diff
@@ -39,7 +39,7 @@ sidebar_is_loaded:
 click_sidebar_overview_tab:
   element:
     selector:
-      xpath: //div[contains(@class,'overview__sidebar')]//button[text()='Overview']
+      xpath: //div[contains(@class,'overview__sidebar')]//button[text()='Details']
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/overview.xyaml]: In 4.4, Overview tab is changed to 'Details'

File: lib/rules/web/admin_console/4.4/common_ui_elements.xyaml [hunk 3]
```diff
@@ -75,6 +75,14 @@ click_tab:
...
     op: click
     timeout: 20
+click_environment_tab:
+  params:
+    tab_name: Environment
+  action: click_tab
+click_overview_tab:
+  params:
+    tab_name: Details
+  action: click_tab
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/common_ui_elements.xyaml]: In 4.4, `Overview` tab is changed to text `Details`

**Comment by SSshahan** [comment]
@pruan-rht Can you help merge the fixes?

**Pull request merged**